### PR TITLE
Update features section in product card i2

### PR DESCRIPTION
**Pull request by monsieur-z**
### Changes proposed in this Pull Request

This PR updates both style and content of the features section, in the plan cards of the new pricing page.

FIxes 1196341175636977-as-1198920526622494

### Implementation notes

- Slide-outs will be integrated in a separate PR.
- The version checks are not ideal, as mentioned in other platforms. Let's think about how to fix this once i2 is released.

### Testing instructions

- Download the PR
- Concurrently run Calypso and Jetpack cloud
- Make sure the variant `v2 - slide outs` is selected in the `jetpackConversionRateOptimization` A/B test
- Visit the plans page in both Calypso and cloud, expand the plans cards (the first 3 cards), and compare them to the mockups (or screenshots below)
- Switch back to the previous variant and check that the page looks like production

#### Mockups
Screen Shot 2020-10-26 at 4 11 50 PM

#### Implementation
Screen Shot 2020-10-26 at 5 26 39 PM

### CODE REVIEW

File: client/lib/plans/features-list.js [hunk 1]
```diff
@@ -1078,17 +1115,27 @@ export const FEATURES_LIST = {
...
-				},
-			} ),
+			getJetpackCROActiveVersion() === 'v2'
+				? i18n.translate( 'Backup {{strong}}{{em}}Real{{nbh/}}time{{/em}}{{/strong}}', {
+						components: {
+							em: <em />,
+							strong: <strong />,
+							nbh: <>&#8209;</>,
+						},
+						comment: '{{nbh}} represents a non breakable hyphen',
```
Comment (elliottprogrammer) [inline client/lib/plans/features-list.js:1132]: Very nice, non-breakable hyphen!  😉

**Review by elliottprogrammer — APPROVED** [review]
Everything LGTM here. Nice 👍

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/5054622

Thank you @monsieur-z for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/5054622).

**Pull request merged**